plugins/modules/postgresql_query.py

```diff
@@ -374,6 +376,12 @@ def main():
     search_path = module.params["search_path"]
     as_single_query = module.params["as_single_query"]
 
+    if path_to_script and as_single_query is None:
+        module.warn('You use the "path_to_script" option with the "as_single_query" '
+                    'option unset. The default is False. '
+                    'To avoid crashes, please read the documentation '
+                    'and define the "as_single_query" option explicitly')
```

Comment:
Since it's no deprecation, it should have a trailing period :)
```suggestion
                    'and define the "as_single_query" option explicitly.')
```